test/integration-test.py

```diff
@@ -195,6 +267,12 @@ def main():
     if not CONFIG_NEXT:
         run_expired_authz_purger()
 
+    # Run the boulder-janitor. This should happen after all other tests because
+    # it will advance the fake clock and delete rows that may otherwise be
+    # referenced by tests otherwise
+    if CONFIG_NEXT:
```

Comment:
This doesn't need to be config-next gated because it doesn't depend on any flags.

Reply:
Good catch, I think I was thinking about requiring migrations only present in config next but the only SQL diffs for the janitor are in the grants run by both sets of configs.

Removed the gating in f7c8a56